src/AmsterdamPHP/Bundle/MeetupBundle/Command/CreateJoindinPlaceholderCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AmsterdamPHP\Bundle\MeetupBundle\Command;
+
+use CL\Slack\Payload\ChatPostMessagePayload;
+use Codeliner\ArrayReader\ArrayReader;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateJoindinPlaceholderCommand
+ */
+class CreateJoindinPlaceholderCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \AmsterdamPHP\Bundle\MeetupBundle\Service\EventService
+     */
+    protected $meetup;
```

Comment:
Is there a reason these properties (like others in the project) are `protected` instead of `private`? Is it a standard i'm not aware of in this project? Just a habbit? Some other reason?